cnxml/__init__.py

```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
+from .validation import *
```

Comment:
flake8 really doesn't like these, but I seem them in __init__ a lot, as way to hide details of implementation of an API. Can we either `# noqa` these, or I suppose I can try tweaking a local .flake8.cfg to ignore them.